makefile

```
@@ -1,6 +1,8 @@
-PHONY: check fmt vet lint test build smoke docker
+.PHONY: check fmt vet lint test build smoke docker verify-metrics grafana-ds-uid
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)